Deepgram/Billing/BillingClient.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System.Net.Http;
+using System.Threading.Tasks;
+using Deepgram.Request;
+
+namespace Deepgram.Billing
+{
+    internal class BillingClient: IBillingClient
+    {
+        private CleanCredentials _credentials;
+
+        public BillingClient(CleanCredentials credentials)
+        {
+            _credentials = credentials;
+        }
+
+        /// <summary>
+        /// Generates a list of outstanding balances for the specified project. To see balances, the authenticated account must be a project owner or administrator
+        /// </summary>
+        /// <param name="projectId">Unique identifier of the project for which you want to retrieve outstanding balances</param>
+        /// <returns>List of Deepgram balances</returns>
+        public async Task<Billing[]> GetAllBalancesAsync(string projectId)
```

Comment:
This return type is incorrect. If you look at the documentation, the API doesn't return an array of `Billing` objects. It instead returns an object with a `balances` property that is an array of `Billing` objects.

```js
{
  "balances": [
    {
      "balance_id": "uuid",
      "amount": 0,
      "units": "string",
      "purchase": "string"
    }
  ]
}
```

I believe we've handled things like that in the past with a "List" object. (e.g. BillingList class with a singular property called Balances that is the array of Billing objects.)